### ISSUE TITLE
WebSocketSharp missing

### ISSUE THREAD

**Issue by reillydonovan** [BenVanCitters/UnityNetworkingExamples#1]
When I open the project I am returned the following error, in addition to others related to websockesharp.  When I go into the websocket plugin folder it is empty.  Any suggestions?  Thanks mang.

Assets/Scripts/WebSockets/WebSocketClient.cs(5,7): error CS0246: The type or namespace name `WebSocketSharp' could not be found. Are you missing an assembly reference?

**Comment by BenVanCitters** [comment]
Good catch - looks like a missing DLL in the plugins folder (probably neglected due to git ignore setting).  I'll push that asap.
